c_sharp_json_diff/JsonDiff.cs

```diff
@@ -1,55 +1,75 @@
-﻿using Newtonsoft.Json;
+﻿using JsonDiffPatchDotNet;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using System;
 using System.Collections;
 using System.Collections.Generic;
 
 namespace c_sharp_json_diff
 {
-    public class Result {
-        public List<Diff> Diffs { get; set; } = new List<Diff>();
-
-    }
-    public class Diff
-    {
-        [JsonProperty("property")]
-        public string jsonProperty { get; set; } = "";
-        [JsonProperty("previous")]
-        public double previous { get; set; }
-        [JsonProperty("current")]
-        public double current { get; set; }
-        [JsonProperty("difference")]
-        public double difference { get; set; }
-    }
     public class JsonDiff
     {
-        public static Result GetDiff(IDictionary<string, double> first, IDictionary<string, double> second)
+        public static JObject NumericDiff(string left, string right)
```

Comment:
There should be a version that only deals with strings (take two strings & return a string). That way the client code doesn't even need to add a dependency on the JSON parser library.